package.json

```diff
@@ -15,26 +15,26 @@
     "test": "jest"
   },
   "dependencies": {
-    "@babel/core": "^7.4.5",
-    "@mdx-js/mdx": "^1.0.19",
-    "@mdx-js/react": "^1.0.16",
-    "babel-jest": "^24.8.0",
-    "babel-preset-gatsby": "^0.2.7",
+    "@babel/core": "^7.8.4",
+    "@mdx-js/mdx": "^1.5.5",
+    "@mdx-js/react": "^1.5.5",
+    "babel-jest": "^25.1.0",
+    "babel-preset-gatsby": "^0.2.29",
     "dotenv": "^8.0.0",
     "eventbrite": "^1.2.1",
-    "gatsby": "^2.1.19",
-    "gatsby-plugin-google-analytics": "^2.0.20",
-    "gatsby-plugin-netlify": "^2.0.17",
-    "gatsby-plugin-offline": "^2.2.4",
-    "gatsby-theme-conference": "^0.0.3",
-    "jest": "^24.8.0",
+    "gatsby": "^2.19.18",
+    "gatsby-plugin-google-analytics": "^2.1.35",
+    "gatsby-plugin-netlify": "^2.1.32",
+    "gatsby-plugin-offline": "^3.0.35",
+    "gatsby-theme-conference": "^0.1.1",
```

Comment:
Build failed because of this one. Whoops!

Reply:
Maybe just leave that one at the latest `0.0.` patch version? It's just the theme right?

Reply:
yea, i agree. I just didn't back it out last night because I was already done by the time I realized this problem.